src/components/profile/profile.component.js

```diff
@@ -15,11 +15,14 @@ import { isClassic } from '../../utils/helpers/style-helper';
 import baseTheme from '../../style/themes/base';
 
 class Profile extends React.Component {
+  /** Returns the hasSrc component */
```

Comment:
Maybe something like this would be better?
```suggestion
  /** Determines whether a `src` prop has been supplied */
```